src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_References/Arrays_Ex3.java

```diff
@@ -0,0 +1,17 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_References;
+
+public class Arrays_Ex3 {
+    public static int indexOfMax(int[] array) {
+        int maxOfElements = array[0];
+        int indexOfMaxNumber = 0;
+        for (int i = 0; i < array.length; i++) {
```

Comment:
Can be started with i = 1, now first element is compared with itself

Reply:
Done, thank you